### PR TITLE
removed old validation

### LINKED ISSUE #44: Remove require validation
Validation for input plugins should not include any validation to check if an input has content in it as that should instead be covered by general validation when the plugin is used in forms.

``` gherkin
Feature: Extraneous Validation
  As a core developer
  I want validation free from a check for content
  So that validation for whether an input is required is controlled by general form validation.

  Scenario: Remove validation
    Given that controlling whether an input should be required is a function of our forms
    When testing whether an input is required or not
    Then that test should included in the validation for this plugin.
```

---

**Extraneous Validation**
- [x] Remove validation

### PR DESCRIPTION
**Pull request by scottnath**
Removes outdated validation

---

Resolves #44 

`DCO 1.1 Signed-off-by: Scott Nath <github@scottnath.com>`

### CODE REVIEW

**Comment by Snugug** [comment]
- Please remove `empty` setting
- Please include validation to ensure it's a valid telephone number. Recommend [validator](https://github.com/chriso/validator.js)'s isMobilePhone check as a good place to start.

**Comment by Snugug** [comment]
https://www.npmjs.com/package/google-libphonenumber may be another good option to look in to

**Comment by scottnath** [comment]
@Snugug should there be a story for that as it is out-of-scope for #44 ?

**Comment by scottnath** [comment]
@Snugug added phone number test regardless of scope.

File: lib/validation.js [hunk 1]
```diff
@@ -18,10 +18,17 @@
  *
  * @module telephoneValidation
  */
+const isMobilePhone = require('validator/lib/isMobilePhone');
 
 module.exports = function telephoneValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
+  const set = settings;
```
Comment (Snugug) [inline lib/validation.js]: Why not just make a locale variable instead of overriding the setting?

**Comment by scottnath** [comment]
@Snugug changes to telephone validation are in.

**Pull request merged**